DEA_datasets/DEA_Waterbodies.ipynb

```diff
@@ -37,7 +37,7 @@
     "* Identify potential water sources for aerial firefighting during bushfires\n",
```

Comment:
<p>Last modified is wrong</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/846/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Fixed.